analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.CSharp12.cs

```diff
@@ -91,22 +72,22 @@ public class DefaultLambdaParameters
     {
         void InvokedFromAnotherLambda()
         {
-            var f1 = (int a, int b) => a + b;
```

Comment:
https://github.com/SonarSource/sonar-dotnet/pull/8093#discussion_r1342442665

These tests were not supposed to be duplicates of the ones for C# 11, but rather tests with the same structure, but having default lambda parameters set, that makes them C# 12 specific.

I have kept the C# 12 testa and added the missing default values to the lambda parameters.

Reply:
Do we need to mention the default lambda parameters in #8072 ?